src/pages/Content/index.js

```diff
@@ -31,7 +31,22 @@ function click(selector) {
 }
 // The function plays a recording when needed
 function playRecording(recordedEvents) {
```

Comment:
Where are you sending back a message that the recording play has finished? (to indicate in Popup).

Reply:
It was removed on purpose as I will implement it in another PR as we have separate task on Trello. Do you want to include it in this PR?

Reply:
Added the sendMessage as requested. I will also update in my task on Trello.